template.yaml

```diff
@@ -187,6 +187,8 @@ Resources:
           "responseLatency":"$context.responseLatency",
           "responseLength":"$context.responseLength"
           }
+      Tags:
+        CheckovRulesToSkip: "CKV_AWS_120"
```

Comment:
Are you able to add comments with the what and why?

Reply:
There is a comment already, line 158 for this one. The comment cannot be in the tag however :(